ChangeLog

```diff
@@ -14,6 +14,7 @@
  * Removed deprecated SORMResult::getEventProbabilityHohenBichler/getGeneralisedReliabilityIndexHohenBichler
  * Removed deprecated OptimizationResult::getLagrangeMultipliers, OptimizationAlgorithm::computeLagrangeMultipliers
  * Removed deprecated FittingTest::Kolmogorov, FittingTest::BestModelKolmogorov (DistributionFactory argument only)
+ * Moved Sample::computeStandardDeviationPerComponent, to computeStandardDeviation
```

Comment:
there is one:
LOGWARN(OSS() << "Sample.computeStandardDeviationPerComponent is deprecated, use computeStandardDeviation");

Reply:
yes but `computeStandardDeviation` changes signature. Thus I think a warn is also mandatory for the `computeStandardDeviation` method

Reply:
We cannot; if someone was using the old signature it will probably throw when using stddev[i,j]
Hopefully the matrix signature was not used in the examples so we can make the assumption it is not scratched much by users

Reply:
ok